test/database_test.go

```diff
@@ -30,15 +24,16 @@ func TestDatabase_UserByEmail(t *testing.T) {
 	}
 
 	// Add a user to find.
-	u := &user.User{
+	username := t.Name()
+	u := &database.User{
 		FirstName: username,
 		LastName:  "Pratchett",
-		Email:     (user.Email)(username + "@pratchett.com"),
+		Email:     (database.Email)(username + "@pratchett.com"),
 	}
 	if err = db.UserCreate(nil, u); err != nil {
 		t.Fatal(err)
 	}
-	defer func(user *user.User) {
+	defer func(user *database.User) {
```

Comment:
this is fine but I guess this is usually done in `teardown` or global init code like `beforeAll`

Reply:
TBH, I can just remove it. The email is unique, so it shouldn't be a problem. And the DB we're using for this doesn't have any persistence, so it won't carry over between runs.